docs/monitoring/08-05-send-notifications.md

```diff
@@ -6,19 +6,19 @@ This tutorial shows you how to configure Alertmanager to send notifications. Ale
 
 ## Prerequisites
 
-This  tutorial is a follow-up of the [**Observe application metrics**](/components/monitoring/#tutorials-observe-application-metrics) and the [**Define alerting rules**](https://kyma-project.io/docs/master/components/monitoring/#tutorials-define-alerting-rules) tutorials that use the `monitoring-custom-metrics` example. Follow this tutorial to deploy the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric and creates an alert based on it. That configuration is required to complete this tutorial.
+This  tutorial is a follow-up of the [**Observe application metrics**](#tutorials-observe-application-metrics) and the [**Define alerting rules**](#tutorials-define-alerting-rules) tutorials that use the `monitoring-custom-metrics` example. Follow this tutorial to deploy the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric and creates an alert based on it. That configuration is required to complete this tutorial.
 
 
 ## Steps
 
 Follow these steps to configure notifications for Slack every time Alertmanager triggers and resolves the `CPUTempHigh` alert.
- 
+
 
 1. Install the Incoming WebHooks application using Slack App Directory.
 
   >**NOTE**: The approval of your Slack workspace administrator may be necessary to install the application.
 
-2. Configure the application to receive notifications coming from third-party services. Read [this](https://api.slack.com/incoming-webhooks#create_a_webhook) document to find out how to set up the configuration. 
+2. Configure the application to receive notifications coming from third-party services. Read [this](https://api.slack.com/incoming-webhooks#create_a_webhook) document to find out how to set up the configuration.
```

Comment:
>...document to find out how to set up the configuration

I'd add _for Slack_ at the end, to highlight that this config is only for one,  exemplary, service, and not general rule